.github/workflows/python-app.yml

```diff
@@ -47,7 +47,7 @@ jobs:
 
     - name: Test with pytest
       run: |
-        python -m coverage run --source=cdci_osa_plugin -m pytest tests -sv --full-trace --log-cli-level=DEBUG  -m 'not dda and not jemx_plugin'
```

Comment:
I changed it so that only dda (backend) requests are excluded. jemx_plugin and isgri_plugin requests, as long as they rely on dummy products, should just work, and no need to be excluded.

Reply:
so, just not `dda` ?

Reply:
yes, since that's what we want to exclude - backend requests which are not feasible from github actions.
But we need to be sure all tests comply with this convention. I only removed to make partial transition.